go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/feast-dev/feast
 
 go 1.17
 
+replace github.com/go-python/gopy v0.4.0 => github.com/feast-dev/gopy v0.4.1-0.20220429180328-4257ac71a4d0
```

Comment:
was the main change to pin it at v0.40.0? was the re-ordering of “replace” needed in go.mod?

Reply:
@pyalex is right, the line change has no effect.

Reply:
Pinning was the first main change. The other main change was in setup.py to pass in the go env to go commands. Without that, the go subprocess would behave unexpectedly since the go config would be different since the process environment isn't inherited by default.